MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -1287,7 +1286,7 @@ namespace WPEFramework {
                                 if (k_ret == 0){
                                     LOGINFO(" %s Termimated\n",script_names[i].c_str());
                                     /*this means we killed the task currently running */
-                                    task_incomplete = true;
+                                    m_task_map[task_names_foreground[i].c_str()]=false;
```

Comment:
m_task_map is being updated from two threads.. plz check on possible race conditions.

Reply:
m_task_map is updated from stopMaintenance only. Test results are attached in the ticket https://ccp.sys.comcast.net/browse/RDKTV-20401?focusedCommentId=15064916&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15064916. Verified both unsolicited and solicited maintenance by invoking stopMaintenance api. No crashes or regressions observed with Maintenance Manager